### PR TITLE
extensions: improve docsting (used in the cli)

### PR DESCRIPTION
**Pull request by sergiusens**
Provide a better explanation of what the goal of this extension is and
what it does when used, providing a much better output for
`snapcraft extension gnome-3-28`.

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

File: snapcraft/internal/project_loader/_extensions/gnome_3_28.py [hunk 1]
```diff
@@ -25,9 +25,28 @@
 
 
 class Gnome_3_28Extension(Extension):
-    """The Gnome extension.
+    """This extension eases creation of snaps that integrate with Gnome 3.28
```
Comment (cjp256) [inline snapcraft/internal/project_loader/_extensions/gnome_3_28.py]: s/Gnome/GNOME?

File: snapcraft/internal/project_loader/_extensions/gnome_3_28.py [hunk 2]
```diff
@@ -25,9 +25,28 @@
...
+
+    It sets up content interface plugs for:
+
+    \b
+    - GTK3 Themes.
+    - Common Icon Themes.
+    - Common Sound Themes
+    - The Gnome runtime libraries and utilities corresponding to 3.28.
+
+    For easier desktop integration, it also sets up each application
```
Comment (cjp256) [inline snapcraft/internal/project_loader/_extensions/gnome_3_28.py]: suggestion: "sets up..." -> "configures each application with the following plugs:"

File: snapcraft/internal/project_loader/_extensions/gnome_3_28.py (hunk 1 above)
Comment (sergiusens) [reply snapcraft/internal/project_loader/_extensions/gnome_3_28.py]: I will leave this one up to @kenvandine but if not comment from him, I do not mind changing

Comment (kenvandine) [reply snapcraft/internal/project_loader/_extensions/gnome_3_28.py]: GNOME is appropriate

**Comment by kenvandine** [comment]
s/Gnome/GNOME/g would be more appropriate.

**Pull request merged**